projects/skaffold.dev/package.yml

```diff
@@ -0,0 +1,38 @@
+# https://skaffold.dev/
+
+distributable:
+  url: https://github.com/GoogleContainerTools/skaffold/tree/v{{version}}
+
+versions:
+  github: GoogleContainerTools/skaffold/tags
+
+platforms:
+  - darwin
+  - linux
+
+dependencies:
+
+build:
+  dependencies:
+    curl.se: '*'
+  working-directory: ${{prefix}}
+  script: |
+    curl -Lfo skaffold curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v{{version}}/skaffold-$PLATFORM
```

Comment:
Does this work correctly? It looks malformed.

Reply:
Good catch! How does the gh ci pass - strange!